cypress/e2e/admin/login.cy.js

```diff
@@ -0,0 +1,45 @@
+import '@testing-library/cypress/add-commands';
+
+describe('Admin Login', () => {
```

Comment:
it can be the correct moment to start implementing PageObjects in tests
ask François about priority about this
if not, these tests are nice